net/mwan3/files/lib/mwan3/common.sh

```diff
@@ -180,3 +175,18 @@ mwan3_count_one_bits()
 	done
 	echo $count
 }
+
+get_uptime() {
+	local uptime=$(cat /proc/uptime)
+	echo "${uptime%%.*}"
+}
+
+get_age() {
+	local time_p time_u
+	iface="$1"
+	time_p="$(cat "$MWAN3TRACK_STATUS_DIR/${iface}/TIME")"
+	[ -z "${time_p}" ] || {
```

Comment:
Do we not have also to the same check as in https://github.com/openwrt/packages/pull/13908/files#diff-512330b586e317486d8128ca7fffafbd5ec1463d9e8de07bdcd2ebb8f770d445R86

Reply:
OFFLINE/ONLINE gets set to zero when the interface is connected/disconnected, but TIME never gets set to zero, so the check does not appear to be needed.

I just took the existing code and moved them into functions, so there should be no change in behavior here.